repos/system_upgrade/el7toel8/actors/sanebackendsmigrate/libraries/library.py

```diff
@@ -0,0 +1,324 @@
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.stdlib import api
+from leapp.models import InstalledRedHatSignedRPM
+
+
+# Database of changes in configuration files of sane-backends
+# between RHELs
+
+"""
+Changes between 1.0.24 and 1.0.27 versions
+"""
```

Comment:
@zdohnal To what this docstring is connected? Is it to all list fom `CANON_DR` to `XEROX_FP`? In such case, the docstring should be below every of those lists, probably little modified to suit to each related list. Or just remove it and keep the comment.

Reply:
It was meant for all those lists, but it does not make sense to do it like that - so I concur on removing.

Reply:
Will be removed in the newest commit.